Source/CarthageKit/Archive.swift

```diff
@@ -58,12 +58,12 @@ private func untargz(archive fileURL: URL, to destinationDirectoryURL: URL) -> S
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
-private let ArchiveTemplate = "carthage-archive.XXXXXX"
+private let archiveTemplate = "carthage-archive.XXXXXX"
```

Comment:
I think, if you really care so much about the naming convention of the top-level constants, then why not follow the best practice and prevent them completely by including them into structs and/or enums instead?